ivy/container/sorting.py

```diff
@@ -23,6 +23,86 @@ def static_argsort(
         *,
         out: Optional[ivy.Container] = None
     ) -> ivy.Container:
+        """
+        ivy.Container instance method variant of ivy.argsort. This method simply wraps the
+        function, and so the docstring for ivy.argsort also applies to this method
+        with minimal changes.
+        
+        Parameters
+        ----------
+        x 
+            input array or container. Should have a numeric data type.
+        axis
+            axis along which to sort. If set to ``-1``, the function must sort along the last axis. Default: ``-1``.
+        descending
+            sort order. If ``True``, the returned indices sort ``x`` in descending order (by value). If ``False``, the returned indices sort ``x`` in ascending order (by value). Default: ``False``.
```

Comment:
this line is too long, please update it so that one line only has up to 88 characters, rest of the content would go to a new line.